client/post-editor/post-editor.jsx

```diff
@@ -97,9 +97,26 @@ const messages = {
 				comment: 'Editor: Message displayed when a post is published privately, with a link to the site it was published on.'
 			} );
 		},
+		scheduled: function() {
+			var site = this.props.sites.getSelectedSite();
+
+			if ( ! site ) {
+				return i18n.translate( 'Post scheduled!' );
+			}
+
+			return i18n.translate( 'Post scheduled on {{siteLink/}}!', {
+				components: {
+					siteLink: <a href={ site.URL } target="_blank">{ site.title }</a>
+				},
+				comment: 'Editor: Message displayed when a post is scheduled, with a link to the site it was published on.'
+			} );
+		},
 		view: function() {
 			return i18n.translate( 'View Post' );
 		},
+		viewPreview: function() {
+			return i18n.translate( 'Preview Post' );
```

Comment:
My rationale was that "View Post" makes me think the post is already live on my site. If I scheduled a post and noticed "View Post," I would think "Oh shoot—did it publish?"

"View Preview" could be a better compromise.